.github/CODEOWNERS

```diff
@@ -33,6 +33,7 @@ go.sum @grafana/backend-platform
 
 # Backend code docs
 /contribute/style-guides/backend.md @grafana/backend-platform
+/contribute/architecture/backend @grafana/backend-platform
```

Comment:
Should we also move `/contribute/architecture/communication.md`, `/contribute/architecture/database.md` and `/contribute/architecture/services.md` in this new backend folder too?

Reply:
@wbrowne I'd be for it, would simplify things and mesh with CODEOWNERS. @torkelo?

Reply:
:+1:

Reply:
I guess @torkelo approves of moving them, I can do it in another PR :)